app/lib/actions/katello/content_view_puppet_module/destroy.rb

```diff
@@ -11,7 +11,7 @@ def plan(repository)
           repository.puppet_modules.each do |puppet_module|
             # first, process content view puppet modules that have been specified by version
             library_repos = ::Katello::Repository.in_environment(repository.organization.library).
-                                                  where(:pulp_id => puppet_module.repoids)
+              where(:pulp_id => puppet_module.repositories.map(&:pulp_id))
```

Comment:
I guess that is a personal preference thing too, then. I think it looks better as it was before, but its up to you as long as it doesn't break rubocop (again, unless other team members disagree)